.github/workflows/merge.yml

```
@@ -196,6 +197,23 @@ jobs:
           NEW_RELIC_LICENSE_KEY: ${{ secrets.NEW_RELIC_LICENSE_KEY }}
 
+  deploy-nginx:
+    needs:
+      - terraform
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v6
+
+      - name: Setup HTTPS
+        uses: ./.github/actions/setup-https
```

actionlint (1.7.12)

[error] description is required in metadata of "Setup HTTPS" action at "/home/jailuser/git/.github/actions/setup-https/action.yml"

(action)